torch_em/model/unetr.py

```diff
@@ -68,10 +73,11 @@ def __init__(
         out_channels: int = 1,
         use_sam_stats: bool = False,
         use_mae_stats: bool = False,
+        resize_input: bool = False,  # TODO should this be the default???
```

Comment:
I think we can make this the default. This could be our logic to upsample the images (ofc from SAM) that we resize along the longest size first, next pad, and then have the images in the expected dimension. (we also need to keep in mind our plans for dynamic input shapes, but we can discuss that later)